app/src/main/java/com/github/se/orator/ui/theme/DimensionsFactory.kt

```diff
@@ -0,0 +1,84 @@
+// File: DimensionsFactory.kt
```

Comment:
This file is a great addition to make our app scale better across all devices. Nice one